frontend/src/interfaces/IPropertyApiModel.ts

```diff
@@ -36,6 +36,9 @@ export interface IPropertyApiModel extends Api_ConcurrentVersion {
   propertyType?: Api_TypeCode<string>;
   status?: Api_TypeCode<string>;
 
+  regionType?: Api_TypeCode<number>;
+  districtType?: Api_TypeCode<number>;
+
```

Comment:
as per previous comment would love to see this named just region, district to align with concept API_Property

Reply:
the problem is that these are not part of the concept, but the old model. We should get rid of it completly